lib/backend/syncersv1/updateprocessors/configurationprocessor.go

```diff
@@ -240,6 +240,9 @@ func (c *configUpdateProcessor) processAddOrModified(kvp *model.KVPair) ([]*mode
 					default:
 						value = fmt.Sprintf("%v", vt.Seconds())
 					}
+				case []string:
+					// Make a list of strings comma delimited
+					value = strings.Join(vt, ", ")
```

Comment:
nit:  do you really need a trailing space?  this is just for internal message flow so prolly not necessary.  don't really mind though provided felix can parse it ok.

Reply:
ya. not really needed and parsing in felix would be fine. will nuke it.